src/comm.c

```diff
@@ -8,39 +8,26 @@
 /*************************************************************************/
 
 /*
- * This file contains all of the OS-dependent stuff:
- *   startup, signals, BSD sockets for tcp/ip, i/o, timing.
- *
- * The data flow for input is:
- *    Game_loop ---> Read_from_descriptor ---> Read
- *    Game_loop ---> Read_from_buffer
- *
- * The data flow for output is:
- *    Game_loop ---> Process_Output ---> Write_to_descriptor -> Write
- *
- * The OS-dependent functions are Read_from_descriptor and Write_to_descriptor.
- * -- Furey  26 Jan 1993
+ * Originally contained a ton of OS-specific stuff.
+ * We only support linux these days...so we stopped pretending...
```

Comment:
RIP my dreams of running the mud on an ebay'd SGI Onyx RealityEngine

Reply:
Sorry mang :P